### ISSUE TITLE
UnhandledPromiseRejectionWarning: ERROR

### ISSUE THREAD

**Issue by KumarAbhirup** [KumarAbhirup/bulk-mail-cli#13]
**Run `bulkmail config --auth` and enter wrong credentials to reproduce** the following errors at commit 1f57cac0ace7ddcb9a1dd97efe2f9ab25e52ddc0.

```
(node:5174) UnhandledPromiseRejectionWarning: ERROR
(node:5174) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5174) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Good luck debugging! 🔢

**Comment by KumarAbhirup** [comment]
FIXED! 👍 💯 🥇

**Comment by KumarAbhirup** [comment]
Fixed at 9e959f2. (v0.0.80) 💯
